index.js

```diff
@@ -28,34 +30,46 @@ async function fetchCharacters() {
     maxPage = data.info.pages;
     console.log(maxPage);
     cardContainer.innerHTML = "";
+    pagination.textContent = `${page} / ${maxPage}`;
     data.results.forEach((character) => {
       const cardElement = CharacterCard(character);
       cardContainer.append(cardElement);
     });
   } catch (error) {
     console.error("Could not fetch data", error);
   }
-  updatePagination();
 }
+
 fetchCharacters();
+const pagination = NavPagination(page, maxPage);
+navigation.append(pagination);
```

Comment:
can be deleted

Reply:
Yeah i got it. I can do it when i do the searchBar refacturing :)